public/controllers/management/components/management/reporting/reporting-table.js

```diff
@@ -129,39 +159,38 @@ class WzReportingTable extends Component {
       color: color,
       title: title,
       text: text,
-      toastLifeTimeMs: time
+      toastLifeTimeMs: time,
     });
   };
 
   async deleteReport(items) {
-    const results = items.map(async (item, i) => {
-      await this.reportingHandler.deleteReport(item.name);
-    });
+    try {
+      const results = items.map(async (item, i) => {
+        await this.reportingHandler.deleteReport(item.name);
+      });
 
-    Promise.all(results).then(completed => {
-      this.props.updateIsProcessing(true);
-      this.showToast('success', 'Success', 'Deleted successfully', 3000);
-    });
+      Promise.all(results).then((completed) => {
```

Comment:
**issue**: if there is some error in this `Promise.all`, the `catch` from the `try/catch` block doesn't capture the error.

**suggestion**: We should use `await`
```js
await Promise.all(results)
this.props.updateIsProcessing(true);
this.showToast('success', 'Success', 'Deleted successfully', 3000);
```